client/src/components/Buttons.js

```diff
@@ -3,13 +3,13 @@ import { Link } from 'react-router-dom'
 import Button from '@material-ui/core/Button'
 import PropTypes from 'prop-types'
 
-export const LinkButton = ({ link, text }) => {
+export const LinkButton = ({ link, text, buttonVariant }) => {
```

Comment:
Property `buttonVariant` doesn't seem to be used anywhere, should it be removed?